bundles/ch.elexis.core.data/src/ch/elexis/data/PersistentObject.java

```diff
@@ -528,7 +528,7 @@ public void settingWritten(String key, String value){
 		} else {
 			if (!locale.toString().equals(dbStoredLocale)) {
 				String msg = String.format(
-					"Your locale [%1s] does not match the required database locale [%2s]. Ignore?",
+					"Your locale [%1s] does not match the required database locale [%2s] as specified in config table. Ignore?",
```

Comment:
Warum brauchen wir das? Diese Meldung ist für den Benutzer gedacht, bringt ihm das irgendeinen
Vorteil zu wissen, dass das in der Config-Tabelle angegeben ist?

Reply:
Weil der Benutzer auch ein Entwickler wie ich sein kann, der den Code nicht genau kennt und dann lange in den Einstellungen der Datenbank sucht und nicht in der config Tabelle!

Reply:
Find ich zwar nicht gut - aber ok